generics/services/users.js

```diff
@@ -261,78 +253,135 @@ const profileReadPrivate = function (userId) {
 //         parentEntities.push(entity.id)
 //     }
 //     });
-    
+
 //     if( parentEntities.length > 0 ){
 //         await getSubEntitiesBasedOnEntityType(parentEntities,entityType,result)
-//     } 
-    
+//     }
+
 //     let uniqueEntities = _.uniq(result);
-//     return uniqueEntities;    
+//     return uniqueEntities;
 // }
 
-
 /**
-  * get user roles data token.
-  * @method
-  * @name getUserRoles
-  * @param {Object} roles - {"roles":"all"} 
-  * @returns {Array} - All user roles
-*/
-const getUserRoles = function (
-    filterData =  "all",
-    projection = "all",
-    skipFields = "none"
-) {
-    return new Promise(async (resolve, reject) => {
-        try {
-            let url = userServiceUrl + CONSTANTS.endpoints.LIST_USER_ROLES 
-            const options = {
-                headers : {
-                    "content-type": "application/json",
-                    "internal-access-token": process.env.INTERNAL_ACCESS_TOKEN,
-                },
-                json : {
-                    query : filterData,
-                    projection : projection,
-                    skipFields : skipFields
-                }
-            };
-            request.get(url,options,requestCallback);
-            let result = {
-                success : true
-            };
-            function requestCallback(err, data) { 
-                if (err) {
-                    result.success = false;
-                } else {
-                    let response = JSON.parse(data.body);
-                    if( response.responseCode === HTTP_STATUS_CODE.ok.code ) {
-                        result["data"] = response.result;
-                    } else {
-                        result.success = false;
-                    }
-
-                }   
-                return resolve(result);
-            }
-            setTimeout(function () {
-                return resolve (result = {
-                    success : false
-                 });
-             }, CONSTANTS.common.SERVER_TIME_OUT);
-
-        } catch (error) {
-            return reject(error);
-        }
-    })
+ * get user roles data token.
+ * @method
+ * @name getUserRoles
+ * @param {Object} roles - {"roles":"all"}
+ * @returns {Array} - All user roles
+ */
+const getUserRoles = function (filterData = 'all', projection = 'all', skipFields = 'none') {
```

Comment:
@Prajwal17Tunerlabs Where are we using this function?

Reply:
commented the unused functions